### DISCUSSION TITLE
Einbindung von Hyper 2000

### DISCUSSION THREAD

**Discussion by iroc-fabi** [z-master42/solarflow#32] [General]
Im Wiki steht im Abschnitt "ggf. für Hyper 2000...", dass man Angaben zum Solarflow 1200 eintragen soll. 
Im Hyper Aufsatz  ist doch alles integriert, da muss kein externer Solarflow angeschlossen werden, oder irre ich mich da?

![image](https://github.com/user-attachments/assets/316e9cc9-af0d-4ff2-9252-762105da67da)

**Comment by z-master42** [comment]
Name und Model kannst reinschreiben was du willst. Das ist nur für die Anzeige in Home Assistant.
